framework/include/arastorage/arastorage.h

```diff
@@ -53,11 +53,16 @@
 
 /**
  * @defgroup ARASTORAGE ARASTORAGE
- * @brief Provides APIs for Lightweight Database
+ * @brief Provides APIs for Arastorage Lightweight Database
```

Comment:
How about below?
@brief Provides APIs for Lightweight Database, Arastorage

Reply:
It looks good, I will change